src/components/BannerHighlight/BannerHighlight.vue

```diff
@@ -77,45 +79,35 @@ export default {
     .text {
         margin: 0;
         font-size: 89px;
-        padding-bottom: 43px;
         font-weight: 300;
     }
 
     .prompt-container {
-        position: absolute;
+        position: relative;
+        margin-top: 10px;
 
         .prompt {
             font-size: 28px;
-            overflow: hidden;
-            transition: opacity 200ms ease;
-            opacity: 0;
             padding: 0 15px 0 0;
             margin: 0;
             padding-left: 8px;
         }
 
         .arrow {
-            position: absolute;
-            left: 8px;
-            top: 50%;
-
-            transform: translateY(-50%);
-            transition: left 200ms ease;
-        }
-
-        &:hover .prompt {
-            opacity: 1;
+            transition: transform 200ms ease;
         }
 
-        &:hover .arrow {
-            left: 100%;
+        @media #{$has-hover} {
+            &:hover .arrow {
+                left: 100%;
+                transform: translateX(30%)
+            }
```

Comment:
The hover state would be the entire element. The arrow moves to indicate that a link can be clicked on,  as the entire element is the link, it should move when hover on it.

Reply:
hover added to entire element